assembl/static2/css/components/video.scss

```diff
@@ -1,3 +1,8 @@
+.video-wrapper {
```

Comment:
Is it responsive? Did you try on mobile?

Reply:
You're right, I changed the coden ow I'm using the bootstrap classes embed-responsive embed-responsive-16by9